packages/perseus-core/src/widgets/free-response/index.ts

```diff
@@ -0,0 +1,18 @@
+import type {PerseusFreeResponseWidgetOptions} from "../../data-schema";
+import type {WidgetLogic} from "../logic-export.types";
+
+export type FreeResponseDefaultWidgetOptions = Pick<
+    PerseusFreeResponseWidgetOptions,
+    "question"
+>;
+
+const defaultWidgetOptions: PerseusFreeResponseWidgetOptions = {
+    question: "",
+};
```

Comment:
What's the purpose of this default? It looks like `question` is required in the widget options, so when would the default ever be used?

Reply:
I was following the pattern I see in other widgets, e.g. [PassageWidget](https://github.com/Khan/perseus/blob/basic-free-response-widget/packages/perseus-core/src/widgets/passage/index.ts#L11). I think it makes sense to have `""` be the default `question` prop value in `FreeResponseEditor`, but I'm not sure how else the `freeResponseLogic` is used. Do you know what else it's used for?